resources/views/checkout/partials/sidebar/summary.blade.php

```diff
@@ -10,8 +10,9 @@
             <span>@{{ item.qty }}x @{{ item.name }}</span>
         </li>
         <li
-            class="last:font-medium"
+            class="flex justify-between last:font-medium [&>*]:flex-1 last:[&>*]:text-right"
```

Comment:
are you sure that last is working properly here? because you are using justify between